detect_variants/vep.cwl

```diff
@@ -25,6 +25,8 @@ inputs:
         inputBinding:
             prefix: "-i"
             position: 1
+    reference:
+        type: File
```

Comment:
Should the reference also be optional? Is it only needed when the `--hgvs` option is set to true in this API version (86)?